mrp_component_operation/models/mrp_component_operation.py

```diff
@@ -10,14 +10,14 @@ class MrpComponentOperation(models.Model):
 
     name = fields.Char(help="Component Operation Reference", required=True)
 
-    source_location_id = fields.Many2one(
+    manufacture_location_id = fields.Many2one(
         "stock.location",
-        "Source Location",
+        "Manufacture Location",
         help="The Location where the components are.",
     )
```

Comment:
You will need to provide a migration script for this field rename

Reply:
Migration script added, Thanks!